### PR TITLE
Add GitHub to integration navigation

### PR DESCRIPTION
**Pull request by shairyar**
This PR adds instructions on how to link an application in AppSignal with its repository at GitHub.

### CODE REVIEW

File: source/application/integrations/github.html.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+title: GitHub
+---
+
+[Github][github] is the best place to share code with friends, co-workers, classmates, and complete strangers. Over six million people use GitHub to build amazing things together.
+
+## Adding Organization issue tracker integrations
+
+Head over to your application's [integration](https://appsignal.com/redirect-to/app?to=integrations) page. Click on the "Configure" button for GitHub integration.
+
+<img src="/assets/images/screenshots/github/connfigure.png" style="max-width: 650px" alt="GitHub AppSignal integration">
```
Comment (matsimitsu) [inline source/application/integrations/github.html.md:11]: configure is written with a typo (connfigure)

File: source/application/integrations/github.html.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+On the next step click on the "Install Github app" button.
+
+<img src="/assets/images/screenshots/github/install.png" style="max-width: 650px" alt="Install GitHub app">
+
+You will be redirected to GitHub, here you should select your account/organization where the application sourcecode exists.
```
Comment (matsimitsu) [inline source/application/integrations/github.html.md:17]:
```suggestion
You will be redirected to GitHub, here you should select your account/organization where the application source code exists.
```

**Review by matsimitsu — CHANGES_REQUESTED** [review]
Nice!

File: source/application/integrations/github.html.md (hunk 1 above)
Comment (shairyar) [reply source/application/integrations/github.html.md]: Fixed

**Review by matsimitsu — APPROVED** [review]

**Pull request merged**